[2022-08-03 00:47:59,061] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.create_table manual__2022-08-03T00:47:53.106948+00:00 [queued]>
[2022-08-03 00:47:59,100] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.create_table manual__2022-08-03T00:47:53.106948+00:00 [queued]>
[2022-08-03 00:47:59,111] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 00:47:59,151] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-03 00:47:59,164] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 00:47:59,380] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_table> on 2022-08-03 00:47:53.106948+00:00
[2022-08-03 00:47:59,826] {standard_task_runner.py:52} INFO - Started process 115 to run task
[2022-08-03 00:47:59,876] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ltv-pipeline', 'create_table', 'manual__2022-08-03T00:47:53.106948+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmp6j_nymml', '--error-file', '/tmp/tmpx2yx_mwp']
[2022-08-03 00:47:59,896] {standard_task_runner.py:80} INFO - Job 15: Subtask create_table
[2022-08-03 00:48:00,994] {task_command.py:371} INFO - Running <TaskInstance: ltv-pipeline.create_table manual__2022-08-03T00:47:53.106948+00:00 [running]> on host b9ebe1eb0954
[2022-08-03 00:48:02,625] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ltv-pipeline
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T00:47:53.106948+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-03T00:47:53.106948+00:00
[2022-08-03 00:48:02,746] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-08-03 00:48:02,780] {dbapi.py:231} INFO - Running statement: 
                CREATE TABLE IF NOT EXISTS users(
                    first_name TEXT NOT NULL,
                    last_name TEXT NOT NULL
                )
        , parameters: None
[2022-08-03 00:48:02,801] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-08-03 00:48:02,898] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ltv-pipeline, task_id=create_table, execution_date=20220803T004753, start_date=20220803T004759, end_date=20220803T004802
[2022-08-03 00:48:03,210] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-03 00:48:03,558] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
